models/service.go

```diff
@@ -33,6 +33,8 @@ type ServiceOverview struct {
 	Labels map[string]string `json:"labels"`
 	// Selector for Service
 	Selector map[string]string `json:"selector"`
+	// ExportTo for Service
```

Comment:
Minor: worth to comment that this ExportTo comes from the Istio annotation to have a better context.

Reply:
removing this line